src/main/java/com/code_intelligence/jazzer/junit/AgentConfiguringArgumentsProvider.java

```diff
@@ -37,7 +37,8 @@ public Stream<? extends Arguments> provideArguments(ExtensionContext extensionCo
     // FIXME(fmeum): Calling this here feels like a hack. There should be a lifecycle hook that runs
     //  before the argument discovery for a ParameterizedTest is kicked off, but I haven't found
     //  one.
-    FuzzTestExecutor.configureAndInstallAgent(extensionContext, fuzzTest.maxDuration());
+    FuzzTestExecutor.configureAndInstallAgent(
```

Comment:
Could you please add a comment why the agent needs to be installed here and in `FuzzTestExtensions`? We discussed this, but I forgot again 😬

Reply:
Good point, done :-)